JsBasics/Conditions_and_loops/multiplication.js

```diff
@@ -0,0 +1,12 @@
+
+
+//1) for
+
+for (let i = 0; i <= 10; i++) { console.log(i * 7) }
+
+//2) while
+
+let count = 0;
+while (count <= 10) { console.log(count*7); count++ }
```

Comment:
Після крапки з комою переходимо на наступний рядок. Так більш читабельно буде:
```suggestion
while (count <= 10) { 
    console.log(count*7);
    count++
}
```

Reply:
+ той самий недолік що і з if - потрібно вивести повністю всю табличку